### PR TITLE
Upgrade flutter version.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

**Review by kenzieschmoll — APPROVED** [review]

File: third_party/packages/ansi_up/lib/ansi_up.dart [hunk 1]
```diff
@@ -304,11 +304,11 @@ enum PacketKind {
...
 
 class _TextPacket {
-  _TextPacket({required this.kind, this.text = '', this.url = ''});
+  _TextPacket({required this.kind});
 
   PacketKind kind;
-  String text;
-  final String url;
+  String text = '';
+  final String url = '';
```
Comment (kenzieschmoll) [inline third_party/packages/ansi_up/lib/ansi_up.dart:311]: why is this change necessary?

Comment (polina-c) [reply third_party/packages/ansi_up/lib/ansi_up.dart:311]: I got warning from analyzer, that the parameter is never used, that is true.

**Comment by polina-c** [comment]
On 2.11.0-0.1.pre Gallery crashes when DevTools connects to it with the exception described here: https://github.com/flutter/flutter/issues/97500

We need to wait for fix to be in a tagged version.
